README.md

```diff
@@ -2,6 +2,19 @@
 
 The purpose of this repo is to aggregate issues filed against the [kubeadm component](https://github.com/kubernetes/kubernetes/tree/master/cmd/kubeadm).
 
+## What is Kubeadm ?
+Kubeadm is a tool built to provide best-practice “fast paths” for creating Kubernetes clusters. It was designed by bootstrapping rather than provisioning kubernetes cluster, it performs the actions necessary to get a minimum viable cluster up and running. 
+
+## Common Kubeadm cmdlets 
+1. **kubeadm init** to bootstrap a Kubernetes control-plane node
+1. **kubeadm join** to bootstrap a Kubernetes worker node and join it to the cluster
+1. **kubeadm upgrade** to upgrade a Kubernetes cluster to a newer version
+1. **kubeadm config** if you initialized your cluster using kubeadm v1.7.x or lower, to configure your cluster for kubeadm upgrade
```

Comment:
Maybe something like `to manager kubeadm config`

Reply:
I think it is better keep what it was, I was aligned to this page :https://kubernetes.io/docs/reference/setup-tools/kubeadm/kubeadm-config/

Reply:
it's the description of `kubeadm config upload`  not `kubeadm config`